Foundation/NSNumberFormatter.swift

```diff
@@ -180,9 +179,17 @@ open class NumberFormatter : Formatter {
         
         set {
             switch newValue {
-            case .none, .ordinal, .spellOut:
+            case .none, .ordinal:
                 _usesSignificantDigits = false
-                
+
+            case .spellOut:
+                // FIXME: Locale identifier necessary to make unum work; cheat by using a "default"
+                // Should determine proper current locale identifier
+                if locale.identifier.length == 0 {
+                    _locale = Locale(identifier: "en_US")
```

Comment:
I wasn't able to find a way to get the system's locale identifier... for me, `CFLocaleGetValue` returns an empty string on `Locale.current`...
Agreed the branch shouldn't be taken if the user specifies a locale with `Locale(identifier:)`, hence the `if locale.identifier.length` check